consul.hcl

```diff
@@ -1,14 +1,20 @@
 description = "Consul is a distributed, highly available, and data center aware solution to connect and configure applications across dynamic, distributed infrastructure."
 binaries = ["consul"]
-source = "https://releases.hashicorp.com/consul/${version}/consul_${version}_${os}_amd64.zip"
+source = "https://releases.hashicorp.com/consul/${version}/consul_${version}_${os}_${arch}.zip"
 sha256-source = "https://releases.hashicorp.com/consul/${version}/consul_${version}_SHA256SUMS"
 
-version "1.10.3" "1.11.1" "1.11.2" "1.11.3" "1.11.4" "1.11.5" "1.9.17" "1.12.0"
-        "1.12.1" "1.12.2" "1.12.3" "1.13.0" "1.13.1" "1.13.2" "1.11.10" "1.13.3" "1.14.0"
-        "1.14.1" "1.14.2" "1.14.3" "1.14.4" "1.15.0" "1.15.1" "1.15.2" "1.14.7" "1.15.3"
-        "1.16.0" "1.16.1" "1.16.2" "1.16.3" "1.17.0" "1.15.8" "1.17.1" "1.17.2" "1.17.3"
-        "1.18.0" "1.18.1" "1.18.2" "1.19.0" "1.19.1" "1.19.2" "1.20.0" "1.20.1" "1.20.2"
-        "1.20.3" "1.20.4" "1.20.5" {
+version "1.9.17" "1.10.3" {
```

Comment:
Version [1.9.17](https://releases.hashicorp.com/consul/1.9.17/) and [1.10.3](https://releases.hashicorp.com/consul/1.10.3/) don't have `darwin_arm64`